gprofiler/utils.py

```diff
@@ -524,3 +524,15 @@ def read_perf_event_mlock_kb() -> int:
 
 def write_perf_event_mlock_kb(value: int) -> None:
     Path(PERF_EVENT_MLOCK_KB).write_text(str(value))
+
+
+def is_pyinstaller() -> bool:
+    """
+    Are we running in PyInstaller?
+    """
+    # https://pyinstaller.readthedocs.io/en/stable/runtime-information.html#run-time-information
+    return getattr(sys, 'frozen', False) and hasattr(sys, '_MEIPASS')
```

Comment:
This file is getting way too big. We need to refactor it (not necessarily in this PR though). A lot of these "utils" functions would be better as classes

Reply:
This `utils.py` file might be hideous but how are classes related here? lol (specifically asking about this 1?)